DXMainClient/DXGUI/Multiplayer/GameLoadingLobbyBase.cs

```diff
@@ -271,8 +271,10 @@ protected virtual void GetReadyNotification()
 
             if (!IsHost && !Players.Find(p => p.Name == ProgramConstants.PLAYERNAME).Ready)
                 sndGetReadySound.Play();
+#if WINFORMS
 
```

Comment:
```suggestion

#if WINFORMS
```

Reply:
This change would result in a trailing empty line when not WinForms, potentially triggering a code style warning:

```C#
protected virtual void GetReadyNotification()
{
    AddNotice("The game host wants to load the game but cannot because not all players are ready!".L10N("UI:Main:GetReadyPlease"));

    if (!IsHost && !Players.Find(p => p.Name == ProgramConstants.PLAYERNAME).Ready)
        sndGetReadySound.Play();

}
```